src/components/CollectionDetailPage/CollectionDetailPage.tsx

```diff
@@ -190,7 +196,7 @@ export default class CollectionDetailPage extends React.PureComponent<Props, Sta
                         />
                       )
                     ) : (
-                      <Button primary compact disabled={!this.canPublish()} onClick={this.handlePublish}>
+                      <Button primary compact onClick={this.handlePublish}>
```

Comment:
is there any reason to remove this check? (`canPublish`) it's letting me publish empty/incomplete collections

Reply:
This is a bug, great catch!